src/manifest/mod.ts

```diff
@@ -274,9 +274,17 @@ export class SlackManifest {
   private assignRunOnSlackManifestProperties(manifest: ManifestSchema) {
     const def = this.definition as ISlackManifestRunOnSlack;
 
-    // Run on Slack Apps do not manage access tokens
-    // This is set by default as false
-    manifest.oauth_config.token_management_enabled = false;
+    // Oauth Config
+    // Run On Slack manage their own tokens
+    // if and only if at least one user scope is included in the manifest
```

Comment:
I'm not sure I understand this-- Adding scopes in the manifest should not require apps to use token management. In general I don't think any ROSI apps needs token management, right? Having token management enabled would mean the app needs to provide an oauth token server, which ROSI apps cannot do.

Reply:
@jpbrabec - This was also something I'm also unsure about, so thanks for weighing in.

> Adding scopes in the manifest should not require apps to use token management.

I get this error when adding user scopes to the manifest and leaving token_management_enabled: false: 
![Screenshot 2023-07-09 at 10 04 52 PM](https://github.com/slackapi/deno-slack-sdk/assets/55667998/434b25d1-f4d5-4f06-b649-311987ff2f66)

Is this error expected?

Reply:
Ah, that's for user scopes specifically (not bot scopes). Hermes apps do not support user scopes *at all* right now, so this error is basically here to say "You can't be a hermes ROSI app and use user scopes at the same time".

Reply:
@jpbrabec are you open to me changing the error to be able to bypass it if Hermes User Delegated Auth Houston FF is in place